.themes/default/source/arcs.html

```diff
@@ -15,8 +15,6 @@ <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slug
       {% include rp.html rp=arc_rp %}
     {% endfor %}
     </ul>
-  {% else %}
-    {% include rp.html rp=rp %}
```

Comment:
Eh, in my opinion, it at least needs to be sorted better.

But sounds fine, leaving design decisions for later.